dime-coding-standards/checklists/tex/sampling_randomization_checklist.tex

```diff
@@ -0,0 +1,190 @@
+                
+            \end{itemize}
+            
+            
+			\item[] \dimeCheckBox{\textbf{The method and rationale for random processes is described in documentation and understandable in code}, including:} 
+            
+            \begin{itemize}
+                
+                \item[]\dimeCheckBox{The overall sampling strategy is clearly documented, such as any characteristics or levels of observation that affect probabilities of inclusion/exclusion from sample.}
+                
+                \item[] \dimeCheckBox{Handling of strata and clusters are implemented as described in the research design, for example, using the procedures described by \texttt{randtreat} in Stata (or equivalent).}
+
+                \item[] \dimeCheckBox{Handling of unequal strata and cluster sizes is clearly specified, appropriate, and justified.}
+
+                \item[] \dimeCheckBox{Probability weights are calculated and stored for each unit, when required.}
+                
+            \end{itemize}
+            
+            \item[] \dimeCheckBox{\textbf{Random processes-generating samples are implemented in a reproducible script}, including:} 
+            
+            \begin{itemize}
+                \item[] \dimeCheckBox{Version settings are explicitly set according to software requirements (in Stata, the \texttt{version} or \texttt{ieboilstart} commands)}
+                \item[] \dimeCheckBox{Seed is set using a unique, random seed generated from an external source (such as \href{http://bit.ly/stata-random}{random.org}, for each random process that is intended to be independent.} 
+                \item[] \dimeCheckBox{Population dataset is sorted uniquely before each independent random process.}
+            \end{itemize}
+
+            \dimeCheckBox{\textbf{The script outputs a resulting dataset of results of the random processes} for each potential sampling group (or treatment arm)} 
+            \begin{itemize}
+			\item[] \dimeCheckBox{If the sampling result is intended to be “finalized” for field use, there is a logic switch to ensure that final data cannot be overwritten by new data. For example:}
+            \begin{verbatim}         
+            if `replace_results` == 1 save `results` , replace
+            else di as err "Results not saved: To replace results, toggle logic switch.
+            \end{verbatim}
+                \item[] \dimeCheckBox{The output dataset is created with corresponding codebooks, value labels, and variable labels describing all results of the random processes.}
+                \item[] \dimeCheckBox{The final sampling results are stable and reproducible}
+            \end{itemize}         
+		\end{itemize}
+
+        \hfill 
+        
+        \item[]{\textbf{Randomization:}}
```

Comment:
```suggestion
        \item[]{\textbf{Random treatment assignment:}}
```
See here how we carefully call this random treatment assignment and just "randomization" is a process in computer science, not a step in research. https://worldbank.github.io/dime-data-handbook/measurement.html#creating-research-design-variables-by-randomization